### PR TITLE
Code Documentation

### PR DESCRIPTION
**Pull request by Naylin15**
Hi, @rafaelje.

This Pull Request in progress adds the code documentation on the all java sources.

Ref: https://github.com/flyve-mdm/flyve-mdm-android-agent/issues/45

### CODE REVIEW

File: app/src/main/java/org/flyve/mdm/agent/core/user/UserModel.java [hunk 1]
```diff
@@ -47,6 +47,18 @@ public UserModel() {
...
+     * Set the object's properties to equal the arguments given
+     * @param string firstName
+     * @param string lastName
+     * @param string language
+     * @param string phone
+     * @param string mobilePhone
+     * @param string phone2
+     * @param string the administrativeNumber
+     * @param string the picture
+     * @param string the emails
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/mdm/agent/core/user/UserModel.java]: String or ArrayList?

Comment (ajsb85) [reply app/src/main/java/org/flyve/mdm/agent/core/user/UserModel.java]: EmailsData appear in your other comments

Comment (Naylin15) [reply app/src/main/java/org/flyve/mdm/agent/core/user/UserModel.java]: I'll fix it now

**Comment by Naylin15** [comment]
Hi, @rafaelje 

This code documentation is finished 

Could you please help me with the reviews?

Thanks

**Comment by Gianfranco97** [comment]
@Naylin15

**Pull request merged**